portal/client/public/qTrackLogo.svg

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Do you need this **qTrack** Logo in this qStudy project?

Reply:
No, I've added it for the moment, till you create the qStudy logo.